apps/condo/domains/notification/adapters/firebaseAdapter.js

```diff
@@ -118,6 +119,32 @@ class FirebaseAdapter {
         return { title, body }
     }
 
+    /**
+     * Firebase rejects push if any of data fields is not a string, so we should convert all non-string fields to strings
+     * @param data
+     */
+    static prepareData (data = {}) {
+        const result = {}
+        const invalidFields = []
+
+        for (const key in data) {
+            if (data.hasOwnProperty(key)) {
+                result[key] = data[key]
+
+                if (!isString(data[key])) {
+                    result[key] = data[key].toString()
```

Comment:
What if `data[key]` is `Number`?
<img width="543" alt="image" src="https://user-images.githubusercontent.com/25384290/162170735-41f82775-a699-45a7-b587-a6f7fd146d05.png">
```suggestion
                    result[key] = String(data[key])
```